Glacticraft no longer runs in intelliJ

### Mod Loader

Fabric

### Version Information

main

### Log or Crash Report

N/A

### Reproduction steps

1. Run any config
2. Observe gradle error
```
2:12:13 PM: Executing tasks ':classes :testClasses'...


> Configure project :
Fabric Loom: 0.9.54
Galacticraft: 5.0.0-prealpha+build.2846e792-modified
Using project based jar storage

> Task :checkLicenseGametest FAILED
1 actionable task: 1 executed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':checkLicenseGametest'.
> License violations were found: src/gametest/java/dev/galacticraft/mod/gametest/JUnit5XMLTestCompletionListener.java, src/gametest/java/dev/galacticraft/mod/gametest/mixin/TestServerMixin.java, src/gametest/java/dev/galacticraft/mod/gametest/test/GalacticraftGameTest.java, src/gametest/java/dev/galacticraft/mod/gametest/test/GalacticraftTestSuite.java, src/gametest/java/dev/galacticraft/mod/gametest/test/machine/CircuitFabricatorTestSuite.java, src/gametest/java/dev/galacticraft/mod/gametest/test/machine/CoalGeneratorTestSuite.java, src/gametest/java/dev/galacticraft/mod/gametest/test/machine/CompressorTestSuite.java, src/gametest/java/dev/galacticraft/mod/gametest/test/machine/ElectricArcFurnaceTestSuite.java, src/gametest/java/dev/galacticraft/mod/gametest/test/machine/ElectricCompressorTestSuite.java, src/gametest/java/dev/galacticraft/mod/gametest/test/machine/ElectricFurnaceTestSuite.java, src/gametest/java/dev/galacticraft/mod/gametest/test/machine/EnergyStorageModuleTestSuite.java, src/gametest/java/dev/galacticraft/mod/gametest/test/machine/MachineGameTest.java, src/gametest/java/dev/galacticraft/mod/gametest/test/machine/RefineryTestSuite.java, src/gametest/java/dev/galacticraft/mod/gametest/test/PipeTestSuite.java, src/gametest/java/dev/galacticraft/mod/gametest/test/WireTestSuite.java

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s
```

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -282,6 +282,8 @@ publishing {
 }
 
 license {
+    // Just throw a fit about this for now and require this later in development
+    ignoreFailures(true)
     setHeader(project.file("LICENSE_HEADER.txt"))
     include("**/dev/galacticraft/**/*.java")
     include("build.gradle.kts")

